ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query.hs

```diff
@@ -22,6 +22,9 @@ module Ouroboros.Consensus.Ledger.Query (
   , queryEncodeNodeToClient
   ) where
 
+import           Cardano.Slotting.Block (BlockNo (..))
+import           Cardano.Slotting.Slot (WithOrigin (..))
+
```

Comment:
I would move these imports next to `Cardano.Binary`. I think usually it is like:
1. 3rd party libraries
2. Cardano-base
3. Ledger imports
4. Ouroboros-network 
5. Ouroboros-consensus

Reply:
Done!